firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -41,10 +41,19 @@ class FirebaseSessions internal constructor(firebaseApp: FirebaseApp) {
   fun greeting(): String = "Matt says hi!"
 
   private fun initiateSessionStart() {
-    // TODO(mrober): Generate a session
-    Log.i(TAG, "Initiate session start")
+    val sessionState = sessionGenerator.generateNewSession()
+    val sessionEvent =
+      SessionEvent(
+        eventType = SESSION_START,
+        sessionData =
```

Comment:
Then along the lines of the above comment, could we add a test that makes sure we populate the session correctly given a sessionState? This will be trivial for now but more important later on.

Reply:
We may want to mock SessionState for this test

Reply:
Done